app/lib/esClient.js

```diff
@@ -33,11 +50,28 @@ async function getGp(id) {
   }
 }
 
+function getHitsResult(result, code) {
+  const hits = result.hits && result.hits.hits;
+  if (hits.length > 1) {
+    log.warning({ hits }, `Warning multiple records returned for ${code}, expecting only one`);
+  }
+  // eslint-disable-next-line no-underscore-dangle
+  return hits.length > 0 ? hits[0]._source : undefined;
+}
+
+async function getGpByOdsCode(odsCode) {
+  const endTimer = esGetGpHistogramOdsCode.startTimer();
+  const result = await client.search(getSearchQuery(odsCode));
```

Comment:
Does this need wrapping in a `try...catch`, as `getGpByChoicesId` is?

Reply:
The getByChoicesId query throws an exception if no results are found. The 'try catch' is used to return undefined in that case, otherwise the error is rethrown. 

No special handling is required for getGpByOdsCode so the try catch is not required.